### ISSUE TITLE
Support replica settings

### ISSUE THREAD

**Issue by quytang** [algolia/algoliasearch-django#205]
I came into this issue when trying to implement the sorting feature.
I need to create an index for each sorting criteria.
```
class ProductIndex(AlgoliaIndex):
    ...
    settings = {
        'attributesToIndex': ['title'],
        'attributesForFaceting': ['country', 'city', 'price', 'status'],
        'replicas': [
            'product_by_title_asc',
            'product_by_title_desc',
            ...
        ]
    }
    index_name = 'product'
```
Unfortunately, it doesn't work and throws the error
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/base.py", line 305, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/base.py", line 356, in execute
output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/site-packages/django/contrib/algoliasearch/management/commands/algolia_reindex.py", line 22, in
 handle
    batch_size=options.get('batchsize', 1000))
  File "/usr/local/lib/python2.7/site-packages/django/contrib/algoliasearch/models.py", line 241, in reindex_all
    self.__client.move_index(self.index_name + '_tmp', self.index_name)
  File "/usr/local/lib/python2.7/site-packages/algoliasearch/client.py", line 282, in move_index
    return self._req(False, path, 'POST', data=request)
  File "/usr/local/lib/python2.7/site-packages/algoliasearch/client.py", line 542, in _req
    return self._transport.req(is_search, path, meth, params, data)
  File "/usr/local/lib/python2.7/site-packages/algoliasearch/transport.py", line 166, in req
    raise e
algoliasearch.helpers.AlgoliaException: cannot move with a primary/replica index as source
```
I also need to config the ranking/custom ranking function for each replica index but couldn't find the way to deal with it.
Thanks

**Comment by PLNech** [comment]
Hi @quytang, thank you for this report and all my apologies for the long response time!

I'm currently exploring this issue and was able to reproduce it, I will keep you updated here as I progress.

**Comment by PLNech** [comment]
I found the bug and fixed it in 39431e0, released in [`1.3.2`](https://github.com/algolia/algoliasearch-django/releases/tag/1.3.2). 

Let me know if your issue persists after you update 🙂

**Comment by quytang** [comment]
thanks @PLNech
